packages/amazonq/scripts/build/copyFiles.ts

```diff
@@ -73,33 +73,35 @@ const tasks: CopyTask[] = [
     },
 ]
 
-async function copy(task: CopyTask): Promise<void> {
+function copy(task: CopyTask): void {
     const src = path.resolve(projectRoot, task.target)
     const dst = path.resolve(outRoot, task.destination ?? task.target)
 
     try {
-        await fs.copy(src, dst, {
+        fs.cpSync(src, dst, {
```

Comment:
We could avoid the formal dependency on `@types/node@22` by casting this to `any`. We normally wouldn't do that, but it may be fine in this case because it avoids confusion, since our packages/ depend on `@types/node@16` (and that's intentional because vscode ships with node 16).

We can add a check to `techdebt.test.ts` to remind us to revisit this when we bump to node 18: https://github.com/aws/aws-toolkit-vscode/blob/6fa3a9e554f4aa1916d01d00ea93497ca7205cc5/packages/core/src/test/techdebt.test.ts#L30-L35


```suggestion
        (fs as any).cpSync(src, dst, {
```

Reply:
I see, so once we update to 18, we will have `@types/node@18` which should allow us to remove the any?

Also, tracking as a follow-up.